{
  try {
    connection.rollback(savepoint);
  }
 catch (  SQLException e) {
    throw new KettleDatabaseException(Messages.getString("Database.Exception.UnableToRollbackToSavepoint"),e);
  }
}
